docker/php/Dockerfile

```diff
@@ -1,4 +1,5 @@
 FROM php:7.0-fpm
 
-RUN docker-php-ext-install pdo_mysql \
-    && docker-php-ext-install json
+COPY www.conf /usr/local/etc/php-fpm.d/www.conf
```

Comment:
I'm curious as to why the RUN statements were removed and the extensions were replaced. Can you share your reasoning?